README.md

```diff
@@ -1,73 +1,31 @@
 # hipSOLVER
 hipSOLVER is a LAPACK marshalling library, with multiple supported backends.  It sits between the application and a 'worker' LAPACK library, marshalling inputs into the backend library and marshalling results back to the application.  hipSOLVER exports an interface that does not require the client to change, regardless of the chosen backend.  Currently, hipSOLVER supports [rocSOLVER](https://github.com/ROCmSoftwarePlatform/rocSOLVER) and [cuSOLVER](https://developer.nvidia.com/cusolver) as backends.
 
-## Installing hipSOLVER
+## Documentation
 
-### Install Pre-built Packages
-Download pre-built packages from [ROCm's package servers](https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html). Release notes are available on the releases tab of the [library's github page](https://github.com/ROCmSoftwarePlatform/hipSOLVER).
-* `sudo apt update && sudo apt install hipsolver`
+For a detailed description of the hipSOLVER library, its implemented routines, the installation process and user guide, see the [hipSOLVER documentation](https://hipsolver.readthedocs.io/en/latest/).
 
-### Build & Install Library Using Script (Ubuntu only)
-The root of this repository has a helper bash script `install.sh` to build and install hipSOLVER on Ubuntu with a single command.  It does not take a lot of options and hard-codes configuration that can be specified through invoking cmake directly, but it's a great way to get started quickly and can serve as an example of how to build/install.  A few commands in the script need sudo access, so it may prompt you for a password.
-* `./install.sh -id` -- build library, build dependencies, and install (-d flag only needs to be passed once on a system).
-* `./install.sh -ic` -- build library, build clients (tests, benchmarks, and samples), and install.
+## Quickstart Build
 
-To see more options, use the help option of the install script.
-* `./install.sh -h`
+To download the hipSOLVER source code, clone this repository with the command:
 
-### Build & Install Library Manually
-For a standard library installation, follow these steps:
+    git clone git@github.com:ROCmSoftwarePlatform/hipSOLVER.git
```

Comment:
Cloning over SSH requires creating a GitHub account and registering your SSH public key. I'd swap that out for `git clone https://github.com/ROCmSoftwarePlatform/hipSOLVER.git`, which can be done without any additional setup.